sanic_jinja2/__init__.py

```diff
@@ -2,18 +2,26 @@
 # -*- coding: utf-8 -*-
 
 from functools import partial
-from sanic.response import html
+import asyncio
+import functools
+from sanic.response import html, HTTPResponse
+from sanic.views import HTTPMethodView
 from jinja2 import Environment, PackageLoader
 from jinja2.ext import _make_new_gettext, _make_new_ngettext
 
 __version__ = '0.5.5'
 
+CONTEXT_PROCESSORS = 'context_processor'
+APP_KEY = 'jinja2_env'
```

Comment:
what's the purpose to define `APP_KEY` here? If its purpose is allows user to customize the name, it should not be here.